www/%username/avatar.spt

```diff
@@ -19,7 +19,7 @@ if request.method == 'POST':
 
     r = participant.update_avatar(src+':')
     if not r:
-        raise response.error(400, _("We were unable to get an avatar for you from {0}.", src))
+        raise response.error(400, _("Could not fetch avatar for you from {0}.", src))
```

Comment:
"We" has no part in understanding this, the error could be no fault of Liberapay. It is not "an" avatar here, since it is the user's avatar.